lib/std/child_process.zig

```diff
@@ -194,6 +194,32 @@ pub const ChildProcess = struct {
         stderr: []u8,
     };
 
+    /// Collect the output from the processes stdout and stderr. Will return once all output
```

Comment:
Should be `process's`

Reply:
? that looks weird to me.. possession with words plural words equates to a hanging apostrophe; `the dogs' bone`, `processes'`

Reply:
Process is singular here.